app/viewmodels/govuk/CheckboxFluency.scala

```diff
@@ -66,21 +69,28 @@ trait CheckboxFluency {
 
     def describedBy(value: String): Checkboxes =
       checkboxes.copy(describedBy = Some(value))
+
+    def withHint(hint: Hint): Checkboxes =
+      checkboxes copy (hint = Some(hint))
   }
 
   object CheckboxItemViewModel {
 
     def apply(
-               content: Content,
+               content: Content = Empty,
```

Comment:
can we add another apply to have divider and content as empty, rather then setting it to empty in this apply